packages/headless/doc-parser/doc-parser.ts

```diff
@@ -274,6 +274,17 @@ const controllers: ControllerConfiguration[] = [
       ],
     },
   },
+  {
+    initializer: 'buildFolding',
+    samplePaths: {
+      react_class: [
+        'packages/samples/headless-react/src/components/folded-result-list/folded-result-list.class.tsx',
```

Comment:
I wasn't sure which file names to use, those seemed most relevant to what it is visually.